portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,176 @@
+
+    <!-- place the cool image hover effect of each (for each) -->
+    <div id="gallery">
+        <h2> Fun Fact Gallery </h2>
+        <p> 
+            Every single picture in this gallery
+            represent a little fun fact about myself.
+        </p>
+        <div class="row"> 
+            <!-- image on Colombia -->
+            <div class="column"> 
+                <div class="content"> 
+                    <img  src="https://media.nomadicmatt.com/colombiaguide.jpg" alt="Mi querida Colombia" >
+                    <h3>Colombia</h3>
+                    <p>
+                        My family and I are origin from Colombia.
+                        Whenever I go to Colombia, I always have 
+                        a lot fun and am always sad to return home.
+                        Vamos Colombia! 
+                    </p>
+                </div>
+            </div>
+
+            <!-- image on videos games -->
+            <div class="column"> 
+                <div class="content"> 
+                    <img  src="https://image-cdn.essentiallysports.com/wp-content/uploads/20200427185830/the-last-of-us-part-2-release-date.jpg" alt="The Last of Us Part II" >
+                    <h3>Video Games</h3>
+                    <p>
+                        I love video games and I have loved them since
+                        I was kid. Some of my favorite 
+                        games are "The Last of Us Part I", "Witcher 3", 
+                        "Persona 5", "Bloodborne", and "Fifa".
+                    </p>
+                </div>
+            </div>
+
+            <!-- image about the cats -->
+            <div class="column"> 
+                <div class="content"> 
+                    <img  src="images/gatos.jpeg" alt="Mis gatos" >
+                    <h3>Felix y Blu my cats</h3>
+                    <p>
+                        Felix y Blu are my two cats. Felix is very social and loves 
+                        people and the attention of human. While Blu is very
+                        shy and does not really like people she does not know.
+                    </p>
+                </div>
+            </div>
+
+            <!-- image about twins talk being a twin -->
+            <div class="column"> 
+                <div class="content"> 
+                    <img  src="images/Twins.jpeg" alt="oh no">
+                    <h3>Being a Twin</h3>
+                    <p> 
+                        My twin sister Isabella and I we are best friends but 
+                        also very different. We are polar opposites of each other.
+                        My sister very arty. I'm very math and sciencey. 
+                    </p>
+                </div>
+            </div>
+
+            <!-- image on Miami where i grew up -->
+            <div class="column"> 
+                <div class="content"> 
+                    <img  src="https://blog-www.pods.com/wp-content/uploads/2019/08/MG_6_1_Miami.jpg" alt="oh no" >
```

Comment:
Rename alt and remove extra space before src

Reply:
oh yea sorry i was using that alt as a test to see if the image were being imported.